### DISCUSSION TITLE
Stats/metrics counters like count user private/public commits, count user private/public repos, count private/public starred

### DISCUSSION THREAD

**Discussion by faustoct1** [community/community#38694] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

suggested/requested this https://github.com/octokit/octokit.js/issues/2347

Possibility of endpoints to get stats / metrics in read-only mode counters like: count user private/public commits, count user private/public repos, count private/public starred can give insights and open a huge opportunity for developers build things to improve developers career.

A read only endpoint app that gets user stats of public/private repos, commits, gists, starred.

There are legitimate use cases to have endpoints for private stats / metrics.

cc @jokernomade

**Comment by ossanna16** [comment]
Hi there @jokernomade and welcome to our community! Thank you for the feedback!

**Reply by jokernomade** [reply]
Thank you

**Comment by jfullstackdev** [comment]
there is already the external service for that, maybe you can simply collaborate at that repo, since it's really known well 

https://github.com/anuraghazra/github-readme-stats

**Reply by faustoct1** [reply]
I'll have a look
